buildcfg.txt

```diff
@@ -2186,12 +2186,12 @@ PINSA0	15	39
 PINSB0	15	39
 PINSC0	15	39
 PINSD0	15	39
-PISFA0  -144	-88
-PISGA0  -136	-105
-PISGB0  -136	-101
-PISGC0  -136	-99
-PISGD0  -135	-93		; Unused
-PISGE0  -132	-89		; Unused
+PISFA0  -144	-69
+PISGA0  -125	-97
+PISGB0  -125	-82
```

Comment:
Consider increasing B y offset to -87 and C to -92, and the flash to -74. The animation looks a lot smoother.